main.py

```
@@ -41,9 +43,38 @@
-def create_wallet():
+def load_or_create_wallet(datadir: str | None):
+    path = datadir or "."
+    keystore_path = os.path.join(path, "keystore.json")
+
+    # Security Warning:
+    # Keys are currently stored in unencrypted raw hex format for minimality.
+    # In a production environment, this file should be encrypted with a "spending password"
+    # so that the private key only lives in memory when decrypted by the user.
+    if os.path.exists(keystore_path):
+        try:
+            with open(keystore_path, "r") as f:
```

Ruff (0.15.20)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)